data/templates/default.latex

```diff
@@ -156,6 +156,7 @@ $endif$
 $if(links-as-notes)$
 % Make links footnotes instead of hotlinks:
 \renewcommand{\href}[2]{#2\footnote{\url{#1}}}
+\DeclareRobustCommand{\href}[2]{#2\footnote{\url{#1}}}
```

Comment:
Not too familiar with DeclareRobustCommand -- if we have this, do we still need the line before with renewcommand?  It's an odd repetition of the definition.

Also, do you think this is preferable to simply adding a `\protect` in front of `\href` in the source?  Might this `\protect` be necessary for some other cases too, besides links-in-notes?